nf_core/lint.py

```diff
@@ -5,12 +5,13 @@
 the nf-core community guidelines.
 """
 
-import click
 import logging
 import os
 import subprocess
 
-import nf_core
+import click
+
+#import nf_core
```

Comment:
Generally, I prefer not to leave in commented out code generally. Can always see it in the git history.

Also, was moving the `click` import line for a reason?

Reply:
I kept `import nf_core` in there for reference, because I wasn't sure what was going on.

The move of `import click` is again a pylint thing that requests to import the standard libraries before any external ones.